bin/update-ips.bash

```diff
@@ -282,7 +296,7 @@ else
   DIG_CLUSTER="wc"
 fi
 S3_ENDPOINT="$(yq_dig "${DIG_CLUSTER}" '.objectStorage.s3.regionEndpoint' '""' | sed 's/https\?:\/\///' | sed 's/[:\/].*//')"
-if [[ "${S3_ENDPOINT}" == "" ]]; then
+if [[ "${S3_ENDPOINT}" == "" || "${S3_ENDPOINT}" == '""' ]]; then
```

Comment:
Is this because the line above? `yq_dig "${DIG_CLUSTER}" '.objectStorage.s3.regionEndpoint' '""'`, if so could we change the last parameter to just be `''` to solve it that way instead?

Reply:
Absolutely, I thought it was like that for a reason but I don't see why. https://github.com/elastisys/compliantkubernetes-apps/pull/1853/commits/317caca72d6c5df1eb1304b8e240d852d33f1e03